app/Console/Commands/WebStatusCheck.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Http;
+use Illuminate\Support\Facades\Log;
+use Carbon\Carbon;
+use Carbon\CarbonInterface;
+
+class WebStatusCheck extends Command
+{
+    protected $signature = 'monitor:website {domains*}';
+    protected $description = 'Monitor specified domains for non-200 responses';
+
+    private const INITIAL_WAIT = 60;
+    private const ALERT_INTERVALS = [300, 120, 60];
+    private const MAX_ALERTS = 7;
+
+    private $domainStates = [];
+
+    public function handle()
+    {
+        $domains = $this->argument('domains');
+
+        // Setup states for domains
+        foreach ($domains as $domain) {
+            $this->domainStates[$domain] = [
+                'firstAlertTime' => null,
+                'alertCount' => 0,
+                'lastAlertTime' => null,
+                'currentWaitIndex' => 0,
+            ];
+        }
+
+        $this->info('Starting monitoring of domains: ' . implode(', ', $domains));
+
+        while (true) {
```

Comment:
So we expect this command always to be running? `while (true)` worries me. I wonder if, instead, we can think of this command as:

1) Check all the domains in the list once, and log the status somewhere
2) Go through the domain statuses and decide if any action needs to be taken on any of them

Then we can think of the code in this script as a single iteration, and have the Kernel below set to run the command every minute.